### ISSUE TITLE
Why not provided log4j？

### ISSUE THREAD

**Issue by Dreampie** [sgroschupf/zkclient#37]
Now，I use logback，must excludes log4j：

``` xml
<dependency>
        <groupId>com.101tec</groupId>
        <artifactId>zkclient</artifactId>
        <version>${zkclient.version}</version>
        <exclusions>
          <exclusion>
            <artifactId>slf4j-log4j12</artifactId>
            <groupId>org.slf4j</groupId>
          </exclusion>
          <exclusion>
            <artifactId>log4j</artifactId>
            <groupId>log4j</groupId>
          </exclusion>
        </exclusions>
      </dependency>
```

**Comment by dchesterman** [comment]
+1 here.  The client should only use the slf4j api to enable consumers to provide their own log impl

**Comment by benmccann** [comment]
Fix here: https://github.com/sgroschupf/zkclient/pull/56

**Comment by dtboctor** [comment]
Is this in a release version that is available now?  we seem to have a dependency on 0.8 of this library, is this fixed in 0.9 or an upcoming release?

**Comment by mspiegel** [comment]
+1 for a fix to this issue

**Comment by jzillmann** [comment]
Just merged #56 to master, if one of you guys could verify everything is as expected i would release that as 0.10 version! Thanks for your patience and interest!
Closing this one...
